http/src/main/resources/leo.conf

```diff
@@ -99,6 +99,7 @@ akka {
     client.idle-timeout = 1 hour
     parsing.max-to-strict-bytes = 128m
     server.parsing.max-content-length = 128m
+    server.parsing.max-uri-length = 16k
```

Comment:
this was just to make authorization work on my BEE, we have already have a PR going in to fix